tomviz/vtkNonOrthoImagePlaneWidget.cxx

```diff
@@ -112,6 +112,7 @@ vtkNonOrthoImagePlaneWidget::vtkNonOrthoImagePlaneWidget()
   this->Reslice->TransformInputSamplingOff();
   this->Reslice->AutoCropOutputOff();
   this->Reslice->MirrorOff();
+  this->Reslice->SetSlabModeToSum();
```

Comment:
Is there a reason to do this here, rather than if the user sets the slab mode?

Reply:
I set the mode to sum because you said the typical use case would be summation, but the slab mode defaults to mean.

Reply:
OK, and I guess that is what is happening with the widgets, makes sense. Thanks.